train.py

```diff
@@ -173,7 +181,8 @@ def get_dataset(dataset_name):
     elif dataset_name == "AGNewsText":
         return AGNewsText
     elif dataset_name == 'R8Graph':
-        return R8Graph(device)
+        tokenizer = RobertaTokenizerFast.from_pretrained("roberta-base")
```

Comment:
Naturally, this function will have to be rewritten but that's not a concern right now